src/common/json.cpp

```diff
@@ -9,11 +9,12 @@ namespace json
     {
         try
         {
-            std::wifstream file(file_name.data(), std::ios::binary);
+            std::ifstream file(file_name.data(), std::ios::binary);
             if (file.is_open())
             {
-                using isbi = std::istreambuf_iterator<wchar_t>;
-                return JsonValue::Parse(std::wstring{ isbi{ file }, isbi{} }).GetObjectW();
+                using isbi = std::istreambuf_iterator<char>;
```

Comment:
While we at it, let's also get rid of `std::istreambuf_iterator`. Something like this should work:
```cpp
    file.seekg(0, file.end);
    auto length = file.tellg();
    file.seekg(0, file.beg);
    std::string obj_str;
    obj_str.resize(length);
    file.read(obj_str.c_str(), length);
```

Reply:
If we want to get rid of `std::istreambuf_iterator` would it be cleaner dump file to `std::ostringstream` and get `std::string` content from there ?

Reply:
that'd be doing the same thing as `std::istreambuf_iterator`, while `.read`/`.write` should perform much faster.

Reply:
@yuyoyuppe Done. Take another look please. Thanks